cmd/cluster-display/main.go

```diff
@@ -77,7 +78,8 @@ func addSchemes() error {
 	return nil
 }
 
-func getClusterPoolPage(ctx context.Context, hiveClient ctrlruntimeclient.Client) (*Page, error) {
+func getClusterPool(ctx context.Context, hiveClient ctrlruntimeclient.Client) ([]map[string]string, error) {
+	logrus.Info("Calling getClusterPool ...")
```

Comment:
If you meant to leave this log in, I think it should be `debug` level

Reply:
Done